baybe/parameters/utils.py

```diff
@@ -87,3 +89,51 @@ def get_parameters_from_dataframe(
 def sort_parameters(parameters: Collection[Parameter]) -> tuple[Parameter, ...]:
     """Sort parameters alphabetically by their names."""
     return tuple(sorted(parameters, key=lambda p: p.name))
+
+
+def activate_parameter(
```

Comment:
I believe we still need `activate_parameters`. 

- When the original bounds of an active parameter is `[0, a]` or `[-a, 0]`. with `a>threshold`, we still need this method to shift the bounds so that it is guaranteed active. I would say especially in this case, the acquisition function tends to have the optimum on the boundary, which is "0". Therefore, `activate_parameter` helps at least for this case.
- When the original bounds of an active parameter is `[a, b]` with value `a<0<b` and the range is larger than the threshold, we cannot really activate the parameter. Only in this case, it may occur that the optimizer drives this parameter to `0`. If it occurs, the minimum cardinality may not hold. This is the reason why we have the check on minimum cardinality and raise a warning if it gets violated.